callisto_core/accounts/validators.py

```diff
@@ -17,6 +17,11 @@ def validate_school_email(email, school_email_domain):
     ]
 
     if input_email_domain not in allowed and not settings.DEBUG:
+        # XXX (lojikil//Stefan Trail of Bits): I just want to raise if we're ok with this
+        # since if a user adds a specific domain that can easily be tied back to them
+        # (say, me@lojikil.com, lojikil.com would be logged here), someone with access to
+        # the logs could tie a report back to a user. Not concerning per se, but something
+        # we should be aware of
```

Comment:
Huh, apparently school emails -are- getting validated? I missed this on my pass through the app, no idea how.

Reply:
Maybe we can do some client-side validation to help prevent folks from entering wrong email domains?